packages/manager/modules/emailpro/src/emailpro.constants.js

```diff
@@ -1,3 +1,5 @@
+export const EMAILPRO_MX_UPGRADE = '#/configuration/email-domain/{{ domain }}/upgrade';
```

Comment:
i think it would be cleaner to export a function instead of a string to interpolate, i don't like the idea that the caller has to perform the interpolation
```javascript
export const EMAILPRO_MX_UPGRADE = (domain) => '#/configuration/email-domain/_domain_/upgrade'.replace('_domain_', domain);
```